tutorials/basic-topics/sorting/bubble-sort.md

```diff
@@ -1,14 +1,156 @@
 ---
+
 title: 'Bubble Sort'
-description: ''
+
+description: 'It is sorting algorithm based on the idea of repeatedly comparing pairs of adjacent elements and then swapping their positions if they exist in the wrong order.'
+
 hide_table_of_contents: true
-draft: true
+
 keywords:
-  - leetcode
-  - tutorial
-  - sorting
-  - algorithm
-  - bubble sort
+
+- leetcode
+
+- tutorial
+
+- sorting
+
+- algorithm
+
+- bubble sort
+
 ---
 
-<TutorialAuthors names="@TBC"/>
+  
+
+<TutorialAuthors  names="@RadhikaChhabra17"/>
+
+  
+
+## Overview
+
+  
+
+Bubble Sort is a sorting algorithm which compares the adjacent elements and swap their positions if they are placed in wrong order. At max, we need to compare adjacent elements for (n-1) iterations where n is the size of array to be sorted. At the end of each iteration, larger(or smaller, as required) value is sorted and placed at correct positions.
```

Comment:
1. Apply LaTex on `n-1`, `n`. e.g. `$n - 1$` and `$n$`
2. add a space after the word `larger`